packages/caliper-core/test/worker/tx-observers/prometheus-manager-tx-observer.js

```diff
@@ -0,0 +1,214 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const mockery = require('mockery');
+const sinon = require('sinon');
+
+/**
+ * simulate Util
+ */
+class Utils {
+    /**
+     *
+     * @param {*} path path
+     * @return {string} the fake path
+     */
+    static resolvePath(path) {
+        return 'fake/path';
+    }
+
+    /**
+     *
+     * @return {boolean} the fake path
```

Comment:
comment doesn't match code

Reply:
Note to self: This needs to be fixed in https://github.com/hyperledger/caliper/blob/c648fa9510fdb856d9cfec455dc8eb42cee2600b/packages/caliper-core/test/worker/tx-observers/prometheus-push-tx-observer.js#L39 and https://github.com/hyperledger/caliper/blob/04a1154fcbc530131b3c03fa97a7ecb833660f56/packages/caliper-core/test/worker/tx-observers/prometheus-tx-observer.js#L39 as well in a later PR